### PR TITLE
Add business page content update tool

### PR DESCRIPTION
**Pull request by hannako**
- Replicates same workflow for updates to business support
page as we currently have for updates to the landing page
- Same signon permissions are currentlty required

[trello](https://trello.com/c/8BLAJXIM/156-publishing-tool-for-business-support-hub-page)

### CODE REVIEW

File: app/controllers/coronavirus_controller.rb [hunk 1]
```diff
@@ -91,4 +117,38 @@ def required_landing_page_keys
...
+      announcements
+      other_announcements
+      guidance_section
+      sections
+      topic_section
+      notifications
+    )
+  end
+
+  def value_object
```
Comment (hannako) [inline app/controllers/coronavirus_controller.rb:137]: wasn't sure what to call this

File: app/controllers/coronavirus_controller.rb [hunk 2]
```diff
@@ -28,24 +69,8 @@ def update_draft
     else
       flash["alert"] = "Error received from GitHub - #{response.code}"
     end
-
-    redirect_to coronavirus_path
-  end
-
-  def publish
```
Comment (hannako) [inline app/controllers/coronavirus_controller.rb:35]: The `publish` method has been renamed `publish_page`, and is called by two controller actions, `publish_landing` and `publish_business`

File: app/controllers/coronavirus_controller.rb [hunk 3]
```diff
@@ -2,23 +2,64 @@ class CoronavirusController < ApplicationController
   before_action :require_coronavirus_editor_permissions!
   layout "admin_layout"
 
-  CONTENT_ID = "774cee22-d896-44c1-a611-e3109cce8eae".freeze
-  CONTENT_URL = "https://raw.githubusercontent.com/alphagov/govuk-coronavirus-content/master/content/coronavirus_landing_page.yml".freeze
-
   def index; end
 
-  def update_draft
```
Comment (hannako) [inline app/controllers/coronavirus_controller.rb:10]: The `update_draft` message has been renamed `fetch_content_and_push` and is called by two controller actions `update_landing` and `update_business`

File: app/controllers/coronavirus_controller.rb (hunk 1 above)
Comment (sihugh) [reply app/controllers/coronavirus_controller.rb:138]: page_details maybe?

**Pull request merged**